### PR TITLE
Clpl 1244 own company details

### PR DESCRIPTION
**Pull request by thackerronak**

### CODE REVIEW

File: cx-portal/src/features/admin/mapper.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { CompanyDetails } from './userApiSlice'
+
+export const companyDetailsToCards = (data: CompanyDetails) => [
+  {
+    head: 'content.organization.company.name',
```
Comment (lavanya-bmw) [inline cx-portal/src/features/admin/mapper.ts:5]: translation is missing in this file for head

File: cx-portal/src/components/pages/Organization/index.tsx [hunk 2]
```diff
@@ -1,11 +1,97 @@
...
+
   return (
     <main>
-      <h2>{t('pages.organization')}</h2>
-      <p>content of the page</p>
+      <PageHeader
+        title={t('pages.organization')}
+        topPage={false}
+        headerHeight={200}
+      ></PageHeader>
```
Comment (lavanya-bmw) [inline cx-portal/src/components/pages/Organization/index.tsx:76]: PageHeader could be a self closing tag here

**Review by lavanya-bmw — CHANGES_REQUESTED** [review]
- Please fix the code smells
- add header template to the newly created files
- Please add your changes to the RELEASENOTES.md file with every PR so that we can have an overview by the end of the sprint.

File: cx-portal/src/features/admin/mapper.ts (hunk 1 above)
Comment (thackerronak) [reply cx-portal/src/features/admin/mapper.ts:5]: yes because this is mapper only, this text get translated while render.

**Comment by thackerronak** [comment]
> * Please fix the code smells
> * add header template to the newly created files
> * Please add your changes to the RELEASENOTES.md file with every PR so that we can have an overview by the end of the sprint.

done

**Review by lavanya-bmw — CHANGES_REQUESTED** [review]
Please add header template for mapper.ts and orgainzation.scss too

File: cx-portal/src/features/admin/mapper.ts (hunk 1 above)
Comment (lavanya-bmw) [reply cx-portal/src/features/admin/mapper.ts:5]: Oh ok

**Comment by thackerronak** [comment]
> Please add header template for mapper.ts and orgainzation.scss too

done

**Review by lavanya-bmw — DISMISSED** [review]
lgtm

**Review by lavanya-bmw — APPROVED** [review]
lgtm

**Pull request merged**